### ISSUE TITLE
PHP 7.2: count(): Parameter must be an array or an object that implements Countable

### ISSUE THREAD

**Issue by acip** [bugsnag/bugsnag-php#557]
### Description
*count(): Parameter must be an array or an object that implements Countable* warning is raised.

### Issue
```php
        $sessions = $this->getSessionCounts();
        $this->setSessionCounts([]);
        if (count($sessions) == 0) {
            return;
        }
```
in bugsnag/bugsnag/src/SessionTracker.php:285
### Environment

* bugsnag-php version: 3.19.0
* PHP version: 7.2

**Comment by GrahamCampbell** [comment]
Thanks for the report. Are you using Bugsnag with the Laravel or Symfony library, or natively? This error can only be caused by the function given to `setStorageFunction`, returning `null`.

**Comment by acip** [comment]
Laravel

**Comment by johnkiely1** [comment]
This has been fixed in version [v3.21.0](https://github.com/bugsnag/bugsnag-php/releases/tag/v3.21.0)
